plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rapid7_intsights
 title: Rapid7 IntSights
 description: Rapid7 IntSights is disrupting external threat intelligence with a combination of human and automated collection, intelligent analysis, and strategic threat hunting that turns the clear, deep, and dark webs into an intelligence resource that any company can deploy
-version: 1.0.0
+version: 1.2.0
```

Comment:
If possible, merge this branch and the action branch together. That way we only have to have one extra version.
Suggestion: make one branch that has both changes and increases to 2.0.0 (due to the type change below) We will review this PR and #1029  separately since they are broken out and then merge the combined branch that you create so that they are released in one go.

Reply:
Agreed, this would be much cleaner than making two immediately subsequent releases.

Reply:
I merged here branch from PR: #1029, so here should be all changes with both branches

Reply:
Thanks. Tagging this for our later group PR review later: should we make this 1.1.0? Don't want to confuse any customers